packages/protocol/contracts/stability/GrandaMento.sol

```diff
@@ -222,46 +440,37 @@ contract GrandaMento is
     emit SpreadSet(newSpread);
   }
 
+  /**
+   * @notice Sets the veto period in seconds.
+   * @dev Sender must be owner.
+   * @param newVetoPeriodSeconds The new value for the veto period in seconds.
+   */
+  function setVetoPeriodSeconds(uint256 newVetoPeriodSeconds) public onlyOwner {
+    vetoPeriodSeconds = newVetoPeriodSeconds;
```

Comment:
shall we safety for a minimum? I'd prevent an (unlikely) fat finger.

Reply:
I was thinking it'd be neat to allow governance to decide they want to be the approver themselves rather than the multisig down the road. Meaning governance would have the ability to:
1. set Governance as the approver
2. set `vetoPeriodSeconds` to be 0 so the proposal can be executed immediately after a governance approval updates it

Thoughts? Tbh it's not a super likely scenario. But I'm not super worried about fat fingers because `setVetoPeriodSeconds` will need to be called via a governance proposal, which will receive a lot of scrutiny